Formula/gdal.rb

```diff
@@ -59,6 +59,7 @@ class Gdal < Formula
   depends_on "python@3.11"
   depends_on "qhull"
   depends_on "sqlite"
+  depends_on "swig" => :build
```

Comment:
Needs to be listed with the build dependencies above. (Squash commits again please, thanks.)

Reply:
Thanks. Fixed and squashed.